api/server.go

```diff
@@ -0,0 +1,125 @@
+package api
```

Comment:
Since the API is not required to run any part of threads, it can live in its own package.